scripts/jenkins/ardana/ansible/roles/input_model_generator/vars/templates/network/compact.yml

```diff
@@ -33,7 +33,7 @@ network_groups:
 
   - name: EXTERNAL-VM
     hostname_suffix: extvm
-    tagged_vlan: false
+    tagged_vlan: true
```

Comment:
Is there a reason for reversing this setting?

Reply:
Yes. I updated the PR & commit description to explain that.

Reply:
So just to confirm this is needed to make the model work in a virtualised environment?